Communitter/src/main/java/com/communitter/api/controller/UserController.java

```diff
@@ -43,6 +43,25 @@ public ResponseEntity<User> updateUserEmail(@P ("id") @PathVariable Long id, Use
     public ResponseEntity<User> updateUserPassword(@P ("id") @PathVariable Long id, UserRequest userInfo){
         return ResponseEntity.ok(userService.updateUserPassword(id, userInfo));
     }
+
+    //Ahmet Ahmet
+    @PutMapping("/update/header/{id}")
+    @PreAuthorize("@authorizer.authorizerForUser(#root,#id)")
+    public ResponseEntity<User> updateUserHeader(@P("id") @PathVariable Long id, @RequestParam String header) {
+        return ResponseEntity.ok(userService.updateUserHeader(id, header));
+    }
+
+    @PutMapping("/update/about/{id}")
+    @PreAuthorize("@authorizer.authorizerForUser(#root,#id)")
+    public ResponseEntity<User> updateUserAbout(@P("id") @PathVariable Long id, @RequestParam String about) {
+        return ResponseEntity.ok(userService.updateUserAbout(id, about));
+    }
```

Comment:
do you plan to update these fields seperately? if so, we should get values from body. otherwise, we should also get them from body and merge those endpoints.